libr/bin/format/pe/pe.c

```diff
@@ -672,7 +672,7 @@ static struct r_bin_pe_export_t* parse_symbol_table(struct PE_(r_bin_pe_obj_t)*
 		}
 	}
 	symctr = 0;
-	if (r_buf_read_at (bin->b, sym_tbl_off, (ut8*) buf, bufsz)) {
+	if (r_buf_read_at (bin->b, sym_tbl_off, (ut8*) buf, bufsz) > 0) {
```

Comment:
the loop below assumes the whole buffer have been read, maybe worth checking with == or chopping the loop below in case

Reply:
I do agree. I actually had done it before, but then I chose to revert the changes because I wanted to do the minimal set of changes.